package.json

```diff
@@ -8,19 +8,19 @@
     "web": "expo start --web"
   },
   "dependencies": {
-    "@config-plugins/react-native-webrtc": "^7.0.0",
-    "@daily-co/config-plugin-rn-daily-js": "0.0.3",
-    "@daily-co/react-native-daily-js": "^0.56.0",
-    "@daily-co/react-native-webrtc": "^111.0.0-daily.2",
+    "@config-plugins/react-native-webrtc": "^8.0.0",
+    "@daily-co/config-plugin-rn-daily-js": "0.0.4",
+    "@daily-co/react-native-daily-js": "^0.59.0",
```

Comment:
We need to wait for:
- @daily-co/react-native-daily-js 0.59.0 to be published.
- @daily-co/config-plugin-rn-daily-js [to be merged ](@daily-co/config-plugin-rn-daily-js) and published.

After that, need to remember to install the dependencies again to update `package-lock.json` before merging this PR.